code/modules/transport/pods/ships.dm

```diff
@@ -276,15 +276,15 @@
 	armor_score_multiplier = 1.7
 	health = 400
 	maxhealth = 400
-	speed = 1
+	speedmod = 1
 	desc = "A smaller version of the experimental Y-series of pods."
 
 ////////gold putt
 /obj/machinery/vehicle/miniputt/gold
 	name = "PyriPutt-"
 	icon_state = "putt_gold"
 	armor_score_multiplier = 0.6
-	speed = 0.2
+	speedmod = 5
```

Comment:
Five seems crazy fast when the racer is only 3.33

Reply:
I can change it if you have suggestions, it should be equivalent to without this PR math-wise